ocs_ci/ocs/elasticsearch.py

```diff
@@ -271,7 +278,8 @@ def cleanup(self):
         log.info("Deleting the es resource")
         self.ocp.delete(yaml_file=self.crd)
         log.info("Deleting the es project")
-        self.ns_obj.delete_project(project_name=self.namespace)
+        # self.ns_obj.delete_project(project_name=self.namespace)
+        self.ocp.delete(yaml_file=self.eck_file)
```

Comment:
As part of the cleanup, you should also delete the PVC created in line 216

Reply:
@keesturam PVC can not be deleted before the pod which use it deleted, and the pod is deleted on line 279 which delete in the end the all project.
when the project deleted, there is no PVC to delete, so in the end, the cleanup delete also the PVC.